EnglishFilter/sections/antiadblock.txt

```diff
@@ -11169,3 +11169,5 @@ wolframclient.net#@#.adsbygoogle
 bigbrothercanada.ca#%#//scriptlet('set-constant', 'video_player_config.players.0.adblock', 'false')
 !+ PLATFORM(ios, ext_android_cb)
 @@||bigbrothercanada.ca^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/127332
+athletics.uwaterloo.ca##[aria-label="Ad Blocker Detected"]
```

Comment:
We have this rule, but it's not in the optimized filter https://github.com/AdguardTeam/AdguardFilters/commit/9ac36a528ad4c14642a59dabf0bfc8f271cc7dea. Fixed directly.

Reply:
@scripthunter7 we also don't add new rules to the end of the file - like near to this https://github.com/AdguardTeam/AdguardFilters/blob/master/EnglishFilter/sections/antiadblock.txt#L742.